app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -266,11 +272,69 @@ class MainActivity : AppCompatActivity(), MainView, HasActivityInjector,
     fun setCheckedNavDrawerItem(@IdRes item: Int) = view_navigation.setCheckedItem(item)
 
     override fun showProgress() {
-        progressDialog = ProgressDialog.show(this, getString(R.string.app_name), getString(R.string.msg_log_out), true, false)
+        progressDialog = ProgressDialog.show(
+            this,
+            getString(R.string.app_name),
+            getString(R.string.msg_log_out),
+            true,
+            false
+        )
     }
 
     override fun hideProgress() {
         progressDialog?.dismiss()
         progressDialog = null
     }
-}
+
+    fun changeLanguage() {
+        val languages = arrayOf("English", "Hindi", "Japanese", "Russian", "Italian")
+
+        val builder = AlertDialog.Builder(this)
+        builder.setTitle("Choose Language")
+        builder.setSingleChoiceItems(languages, -1) { dialog, which ->
+            when(which){
+                0->{
```

Comment:
We have other languages support. What about including them?

Reply:
Yes, should I include all of them?

Reply:
@filipedelimabrito Updated! I have added Chinese, Portuguese(BR) and Portuguese(PT) as they are almost translated as of now. I will open a parent issue regarding translation of strings.